sdk/keyvault/azcertificates/client.go

```diff
@@ -0,0 +1,1568 @@
+	if l == nil {
+		return &generated.KeyVaultClientGetCertificateVersionsOptions{}
+	}
+
+	return &generated.KeyVaultClientGetCertificateVersionsOptions{
+		Maxresults: l.MaxResults,
+	}
+}
+
+// ListCertificateVersionsPage contains the current page from a ListCertificateVersionsPager.PageResponse method
+type ListCertificateVersionsPage struct {
+	// READ-ONLY; A response message containing a list of certificates in the key vault along with a link to the next page of certificates.
+	Certificates []*CertificateItem `json:"value,omitempty" azure:"ro"`
+
+	// RawResponse contains the underlying HTTP response.
+	RawResponse *http.Response
+}
+
+// create ListKeysPage from generated pager
+func listKeyVersionsPageFromGenerated(i generated.KeyVaultClientGetCertificateVersionsResponse) ListCertificateVersionsPage {
+	var vals []*CertificateItem
+	for _, v := range i.Value {
+		vals = append(vals, &CertificateItem{
+			Attributes:     certificateAttributesFromGenerated(v.Attributes),
+			ID:             v.ID,
+			Tags:           convertGeneratedMap(v.Tags),
+			X509Thumbprint: v.X509Thumbprint,
+		})
+	}
+
+	return ListCertificateVersionsPage{
+		RawResponse:  i.RawResponse,
+		Certificates: vals,
+	}
+}
+
+// ListCertificateVersions lists all versions of the specified certificate. The full certificate identifer and
+// attributes are provided in the response. No values are returned for the certificates. This operation
+// requires the certificates/list permission.
+func (c *Client) ListCertificateVersions(certificateName string, options *ListCertificateVersionsOptions) ListCertificateVersionsPager {
+	return ListCertificateVersionsPager{
+		genPager: c.genClient.GetCertificateVersions(
+			c.vaultURL,
+			certificateName,
+			options.toGenerated(),
+		),
+	}
+}
+
+// CreateIssuerOptions contains the optional parameters for the Client.CreateIssuer function
+type CreateIssuerOptions struct {
+	// Attributes of the issuer object.
+	Attributes *IssuerAttributes `json:"attributes,omitempty"`
+
+	// The credentials to be used for the issuer.
+	Credentials *IssuerCredentials `json:"credentials,omitempty"`
+
+	// Details of the organization as provided to the issuer.
+	OrganizationDetails *OrganizationDetails `json:"org_details,omitempty"`
+}
+
+func (c *CreateIssuerOptions) toGenerated() *generated.KeyVaultClientSetCertificateIssuerOptions {
+	return &generated.KeyVaultClientSetCertificateIssuerOptions{}
+}
+
+// CreateIssuerResponse is the response struct for the Client.CreateIssuer function
+type CreateIssuerResponse struct {
+	IssuerBundle
+	// RawResponse contains the underlying HTTP response.
+	RawResponse *http.Response
+}
+
+// CreateIssuer adds or updates the specified certificate issuer. This operation requires the certificates/setissuers permission.
+func (c *Client) CreateIssuer(ctx context.Context, issuerName string, provider string, options *CreateIssuerOptions) (CreateIssuerResponse, error) {
```

Comment:
This doesn't appear to be close to https://github.com/Azure/azure-sdk-for-net/blob/a7743d7564c9d60dff0ec17822af0f18323f5f15/sdk/keyvault/Azure.Security.KeyVault.Certificates/src/CertificateClient.cs#L1131 or its related types like [CertificateIssuer](https://github.com/Azure/azure-sdk-for-net/blob/main/sdk/keyvault/Azure.Security.KeyVault.Certificates/src/CertificateIssuer.cs).

Reply:
I believe the `IssuerBundle` and `CertificateIssuer` are essentially the same type. I'm changing the name of IssuerBundle, to CertificateIssuer

Reply:
None of the *Bundle model names made it into the track 2 SDKs.